docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -770,14 +789,18 @@ supports proximity based on geospatial location (i.e., nodes within a given radi
 
 **Usage Notes:**
 
-- Radius can be specified in the following metric units:
+- Radius can be specified in the following metric units. The terms recognized by Storm are listed in parentheses. For
+  example, radius can be specified as `2 km` or `0.5 meters` or `6 feet`:
```

Comment:
We do allow a space between the number/unit when specifying distance, but in most places that would require using quotes around it like `:radius="2 km"` as `:radius=2 km` would be a syntax error. (It looks less nice without the space but just wondering if this could lead to a potential error for a user).

Reply:
Clarified. The problem w/testing when Optic just handles those spaces for you...